### DISCUSSION TITLE
[QUESTION]:频繁open,close file、ilogtail.LOG日志量过大

### DISCUSSION THREAD

**Discussion by 7y-9** [alibaba/ilogtail#1270] [Help] [answered]
使用场景：k8s宿主机部署ilogtail 采集容器挂载在宿主机上的日志
遇到问题：日志量过大，单台机器一小时近【100W】条。大概两三分钟轮转一次
版本：1.8.1
流量：10MB/s
配置量：300+
内核日志：[ilogtail.1.LOG](https://github.com/alibaba/ilogtail/files/13713375/ilogtail.1.LOG)
采集配置：
[ilogtail_config.json](https://github.com/alibaba/ilogtail/files/13713473/ilogtail_config.json)

大概关键字 比例这样：
open file succeeded               20%
close file succeeded, project 20%
close the file:current file has been read, and is marked deleted or the relative container has been stopped          20%
update the real file path of the log reader during closing  20%

想让大佬们帮忙看看频繁的open close是否正常，以及我这种使用场景是否可行。
还有后面两种日志明显不是轮转的场景 为啥会打这两个log。

**Comment by yyuuttaaoo** [comment, marked as answer]
两三分钟轮转一次，是指ilogtail.LOG?
容器场景下，open close比较多的问题可能是容器变化频繁引起的，容器变化后ilogtail会重新加载配置，此时会重新open close一番。如果是这个问题，可以耐心等待一下，我们后面有计划进行优化。

**Reply by 7y-9** [reply]
对的。两分钟就打满了20MB
我们是 ilogtail在宿主机上跑，采集挂载出来的日志文件。这种算不算容器场景?
